### PR TITLE
A simple publish to NPM script

### PR DESCRIPTION
**Pull request by anton-bannykh**
Projects are configurated automatically.
Run `validate` to check the configuration.
Mock NPM registry testing is not ready yet. Use -PdryRun for now.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            download = true
+            workDir = file("${rootProject.buildDir}/nodejs")
+        }
+
+        repositories {
+            mavenCentral()
+        }
+
+        dependencies {
+            compile "org.jetbrains.kotlin:kotlin-stdlib-js"
```
Comment (bashor) [inline build.gradle]: Shouldn't we specify version explicitly?

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        }
+
+        compileKotlin2Js {
+            kotlinOptions {
+                moduleKind = "commonjs"
```
Comment (bashor) [inline build.gradle]: let's switch to `umd`

**Review by bashor — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (anton-bannykh) [reply build.gradle]: Done

File: build.gradle (hunk 2 above)
Comment (anton-bannykh) [reply build.gradle]: Done

**Review by bashor — APPROVED** [review]

**Comment by bashor** [comment]
Thanks!

**Pull request merged**